lib/oxidized/model/fortios.rb

```diff
@@ -46,7 +48,7 @@ class FortiOS < Oxidized::Model
 
 cfg << cmd('end') if @vdom_enabled
 
-    cfg << cmd('show')
+    cfg << cmd('show full-configuration')
```

Comment:
Does this work across all fortios versions?

Reply:
Yes... 
The main difference is show command display only custom configuration parameter, the show full-configuration display all parameter include defaults.

I prefer the show full